Instagram/Instagram/StoriesViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  StoriesViewController.swift
+//  Instagram
+//
+//  Created by Александр Андреевич Щепелин on 17.10.2022.
+//
+
+import UIKit
+
+/// Стартовый экран приложения
+final class StoriesViewController: UIViewController {
+    
+    // MARK: - IB Outlets
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    // MARK: - Private enum
+    private enum Constants {
+        static let identifierCells = "StoriesCell"
+        static let identifierPosts = "PostCell"
+        static let identifierRecommend = "RecommendCell"
+    }
+    
+    // MARK: - Private properties
+    private let identifierStories = Constants.identifierCells
```

Comment:
```suggestion
    private let identifierStories = Constants.identifierCells
```
все это тоже константы добавь все в 1 enum